modules/ts/misc/table_formatter.py

```diff
@@ -4,6 +4,7 @@
 import sys, re, os.path, cgi, stat, math
 from optparse import OptionParser
 from color import getColorizer, dummyColorizer
+import six
```

Comment:
'six' package is not a builtin package (not bundled with Python)

Reply:
fixed!